terraform/remote_data.tf.prod

```diff
@@ -0,0 +1,7 @@
+data "terraform_remote_state" "dev" {
+  backend = "gcs"
+  config = {
+    bucket = "discovery-etcd-io-dev-infrastructure"
```

Comment:
I prefer pattern: `<project>-infrastructure`. It is easy to remember and simpler.